app/views/orders/set.html.erb

```diff
@@ -2,18 +2,62 @@
 
 <%= bootstrap_form_for work_order, url: wizard_path, method: :put  do |f| %>
 
+<table class="table table-striped table-condensed" id="set-list-table">
+  <tr>
+    <thead>
+      <th></th>
+      <th>Submission Name</th>
+      <th>Size</th>
+      <th>Created</th>
+      <th>ID</th>
+    </thead>
+  </tr>
+
   <% if (work_order.set_uuid) %>
-    <%= f.collection_radio_buttons(:original_set_uuid, [work_order.original_set],
-         :id, lambda { |s| s.name+" (size "+work_order.num_samples.to_s+"): "+s.uuid },
-          label: 'Selected set') %>
-  <% else %>
 
-      <%= f.collection_radio_buttons(:original_set_uuid, get_all_aker_sets,
-         :id, lambda { |s| s.name+" (size "+s.meta["size"].to_s+"): "+s.uuid },
-          label: 'Please select a set') %>
+    <% [work_order.original_set].each do |s| %>
+    <tr>
+      <td>
+        <%= f.radio_button(:original_set_uuid, s.id) %>
+      </td>
+      <td>
+        <%= s.name %>
+      </td>
+      <td>
+        <%= pluralize(work_order.num_samples, "Sample") %>
+      </td>
+      <td>
+        <%= time_ago_in_words(s.created_at) + " ago" %>
+      </td>
+      <td>
+        <%= s.uuid %>
+      </td>
+    </tr>
+    <% end %>
 
-  <% end %>
+  <% else %>
 
-  <%= render 'buttons', work_order: work_order, f: f %>
+    <% (get_all_aker_sets.sort_by &:created_at).reverse!.each do |s| %>
+    <tr>
+      <td>
+        <%= f.radio_button(:original_set_uuid, s.id) %>
+      </td>
+      <td>
+        <%= s.name %>
+      </td>
+      <td>
+        <%= pluralize(s.meta["size"], "Sample") %>
+      </td>
+      <td>
+        <%= time_ago_in_words(s.created_at) + " ago" %>
+      </td>
+      <td>
+        <%= s.uuid %>
+      </td>
+    </tr>
+    <% end %>
 
```

Comment:
There are 2 blocks of html that are generated almost the same way. I would refactor it a little, unless you think it would make things more complicated, for example, with a helper method that yields the corresponding values (uuid, name, size, created) to the view.